### PR TITLE
Fixed snapper submission

### PR DESCRIPTION
**Pull request by mvidner**
[bsc#973607](https://bugzilla.suse.com/show_bug.cgi?id=973607)

### CODE REVIEW

**Comment by mvidner** [comment]
https://ci.opensuse.org/job/snapper-master/11/console

File: package/rubygem-packaging_rake_tasks.changes [hunk 1]
```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Fri Apr  1 12:16:56 UTC 2016 - mvidner@suse.com
+
+- Fixed snapper submission (bsc#973607)
+  - check:changelog only before osc:sr
```
Comment (jreidinger) [inline package/rubygem-packaging_rake_tasks.changes]: does the change really do it? for me it looks like, check:syntax and then do "osc:commit" that have its own checks. So actually it looks like it is runned even later. If you want to run it only after osc:commit, then add it after it, not before.

**Comment by jreidinger** [comment]
@mvidner even if current version works, I worry it has logic problem that depends on rake implementation. For me check should be done after commit, otherwise it can happen in future, that check is run without commit dependencies.

**Comment by jreidinger** [comment]
LGTM

**Comment by mvidner** [comment]
(lgtm'd after we've clarified in person that `check:changelog` does depend on `package`)

**Pull request merged**